validator/rpc/gateway/gateway.go

```diff
@@ -109,7 +109,7 @@ func (g *Gateway) Stop() error {
 
 func (g *Gateway) corsMiddleware(h http.Handler) http.Handler {
 	c := cors.New(cors.Options{
-		AllowedOrigins:   g.allowedOrigins,
+		AllowedOrigins:   append(g.allowedOrigins, "http://localhost:4200"),
```

Comment:
Do you think that hard-coding "http://localhost:4200" can't be avoided (generally, we either wrap those values in constants, configs, or params)?